prediction_market_agent/agents/safe_guard_agent/whitelist.py

```diff
@@ -0,0 +1,81 @@
+from prediction_market_agent_tooling.gtypes import ChecksumAddress
+from prediction_market_agent_tooling.markets.omen.omen_constants import (
+    SDAI_CONTRACT_ADDRESS,
+    WRAPPED_XDAI_CONTRACT_ADDRESS,
+)
+
+
+def is_whitelisted(address: ChecksumAddress) -> bool:
```

Comment:
Whitelisted addresses won't be verified (wxdai, sdai, Safe contracts, ...), to speed up check and make it cheaper and be less rate limited